modules/core/src/parallel_impl.cpp

```diff
@@ -382,6 +382,13 @@ class ParallelJob
 };
 
 
+// Disable thread sanitization check when CV_USE_GLOBAL_WORKERS_COND_VAR is not
+// set because it triggers as the main thread reads isActive while the children
+// thread writes it (but it all works out because a mutex is locked in the main
+// thread and isActive re-checked).
+#if !defined(CV_USE_GLOBAL_WORKERS_COND_VAR) && defined(__has_feature) &&  __has_feature(thread_sanitizer)
+__attribute__((no_sanitize("thread")))
```

Comment:
> `defined(__has_feature) &&  __has_feature(thread_sanitizer)`

Perhaps this should be split on multiple nested `#if` conditions.
Probably this should be guarder by `defined(__clang__)` too.

---

It makes sense to add link to the issue.

Reply:
Done.